updates based on feedback (#319)
makerdao/governance-manual: parameter-index/surplus-auction/param-surplus-slippage-tolerance.md

>**Technical Docs:** N/A  

## Description
The Smart Burn Engine Slippage Tolerance or `want` parameter controls the amount of deviation between the oracle price of MKR and the price obtained by the Smart Burn Engine. It is one of the parameters that defined whether or not the Smart Burn Engine can be triggered.

The Smart Burn Engine can be triggered when:  

> _Purchase Price >= Oracle Price * Smart Burn Engine Slippage Tolerance (`want`)_

Reviewer: @patrick-j-govalpha I think it's as above since the purchase price (of MKR) can be higher than oracle price by at most 1/want

> _Purchase Price >= Oracle Price * Smart Burn Engine Slippage Tolerance (`want`)_

Author: you multiply oracle price by `want` and by the lot and if the amount of MKR being purchased is higher than the value output from that formula then the transaction succeeds

{% hint style="info" %} 

**Example**

*Maximum System Surplus* = 50 million DAI  

> *Smart Burn Engine Lot Size* = 10,000 DAI

Reviewer: ```suggestion
*Smart Burn Engine Lot Size* = 10,000 DAI  
```

*Smart Burn Engine Slippage Tolerance* = 0.98

> 1. MKR/DAI price in the Uniswap pool is 3% worse than oracle price.

Reviewer: ```suggestion
1. MKR/DAI price according to the oracle is 3% lower than the Uniswap price.
2. Keeper A attempts to trigger the Smart Burn Engine, but the transaction fails.
3. MKR/DAI price according to the oracle is 2% lower than the Uniswap price.
4. Keeper A successfully triggers the Smart Burn Engine and exchanges 10,000 DAI for MKR.
```

Reviewer: Minor edit just to make the percentages clearer

2. Keeper A attempts to trigger the Smart Burn Engine, but the transaction fails.

> 3 .MKR/DAI price in the Uniswap pool is 2% away from oracle price.

Reviewer: ```suggestion
```

>**Technical Docs:** N/A  

## Description
The Smart Burn Engine Slippage Tolerance or `want` parameter controls the amount of deviation between the oracle price of MKR and the price obtained by the Smart Burn Engine. It is one of the parameters that defined whether or not the Smart Burn Engine can be triggered.

The Smart Burn Engine can be triggered when:  

_Purchase Price >= Oracle Price * Smart Burn Engine Slippage Tolerance (`want`)_

{% hint style="info" %} 

**Example**

*Maximum System Surplus* = 50 million DAI
*Smart Burn Engine Lot Size* = 10,000 DAI
*Smart Burn Engine Slippage Tolerance* = 0.98

1. MKR/DAI price according to the oracle is 3% lower than the Uniswap price.
2. Keeper A attempts to trigger the Smart Burn Engine, but the transaction fails.
3. MKR/DAI price according to the oracle is 2% lower than the Uniswap price.
4. Keeper A successfully triggers the Smart Burn Engine and exchanges 10,000 DAI for MKR.

> 2. Keeper A attempts to trigger the Smart Burn Engine, but the transaction fails.

Reviewer: ```suggestion
```
